### ISSUE TITLE
Feature: she endpoint.url somewhere in the ui

### ISSUE THREAD

**Issue by vanhoutenbos** [TwiN/gatus#555]
### Describe the feature request

Hi!

It would be nice to be able to view the exact url the endpoint is pinging, not just the domain.
For example on the endpoint page it would be nice to have an href text with the exact url and being able to click it!

If possible I would love to help adding this feature! 

### Why do you personally want this feature to be implemented?

Currently I go into the config and copy the link from there but that's not always an option and requires extra effort

### How long have you been using this project?

Quite a while

**Comment by TwiN** [comment]
Thing is, while adding a configuration in `endpoints[].ui` to show the full url is a good idea, it wouldn't provide a consistent experience.

For instance, what if the endpoint requires a specific header? Or what if the endpoint has a request body? What if there's a lot of query parameters, which would force us to trim (at least visually) the URL?

Instead, I'm going to suggest an alternative, but it would be contingent on the 2nd implementation suggestion I made [here](https://github.com/TwiN/gatus/issues/106#issuecomment-1703926999) being implemented. If it is implemented, then you could add a menu element of type `curl` that would allow you to generate the curl needed to query the endpoint locally.

Of course, it would be a per-endpoint configuration (since there may be some private endpoints you don't want people to be able to generate a curl command for), but that'd be an option.

**Comment by vanhoutenbos** [comment]
This sounds like a good solution, I understand the challenge with all the options you have in the tool! Most monitoring we do are quite basic (http call to an endpoint and waiting for 200 OK)

**Comment by maxexcloo** [comment]
Would love this feature also - I use gatus as status and also a list of my services so links would be awesome (or even just an option to add a manual link).
